sdks/java/io/hadoop-format/build.gradle

```diff
@@ -90,7 +93,6 @@ dependencies {
   testCompile library.java.cassandra_driver_mapping
   testCompile "org.apache.cassandra:cassandra-all:3.11.8"
   testCompile library.java.postgres
-  testCompile "org.apache.logging.log4j:log4j-core:$log4j_version"
```

Comment:
nit: remove `log4j_version` definition as well since it's used only here.

Reply:
done!